tasmota/include/tasmota_types.h

```diff
@@ -730,8 +730,8 @@ typedef struct {
 #endif  // ESP32
   uint16_t      artnet_universe;           // 734
   uint16_t      modbus_sbaudrate;          // 736
-
-  uint8_t       free_esp32_738[5];         // 738
+  uint16_t      shutter_motorstop;         // 738
+  uint8_t       free_esp32_73A[3];         // 73A
```

Comment:
By choosing this Setting location you are using part of the Settings area once used by other released Settings.

You can still use this BUT you will need to add value reset code to make the variable a default value.

Reply:
The last fix reset anything above 64000. I will submit an additional change to limit it to 5000. this is physically the longest time I can imagine you should wait before moving the shutter again.